corehq/apps/accounting/tests/test_autopay.py

```diff
@@ -19,6 +20,7 @@
 from corehq.apps.accounting.tests.test_invoicing import BaseInvoiceTestCase
 
 
+@attr(slow=700)  # almost certainly migrations
 class TestBillingAutoPay(BaseInvoiceTestCase):
```

Comment:
Is this test associated with migrations because it is the first test to run after migrations are applied? That seems pretty fragile—I think adding a new test with a module path that sorts alphabetically before this one would break it. Would be great if we could time the migrations separately so their time does not get added to some random test.

Reply:
Agreed that this is fragile. I didn't want to complicate the changes to the plugin too much. Another issue with this is that I have no idea how fast these are.

Another way to work around this could be to proactively create that test case with a slow attr?